packages/fxa-payments-server/src/routes/Subscriptions/index.stories.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import MockApp from '../../../.storybook/components/MockApp';
+import { Subscriptions, SubscriptionsProps } from './index';
+
+function init() {
+  storiesOf('routes/Subscriptions', module)
+    .add('blank slate', () => (
+      <MockApp>
+        <Subscriptions {...baseProps} />
+      </MockApp>
+    ))
+    .add('loading', () => (
+      <MockApp>
+        <Subscriptions {...{
+          ...baseProps,
+          subscriptions: {
+            loading: true,
+            error: false,
+            result: null
+          }
+        }} />
+      </MockApp>
+    ))
+    .add('error', () => (
```

Comment:
I suppose `[object Object]` here isn't a bug in the Storybook, just in code that Storybook is calling...

![image](https://user-images.githubusercontent.com/44390/58832819-65b05180-8615-11e9-8654-7ebd182b7b14.png)

Reply:
Yeah, this is just what the component does right now. Planning to change this when implementing error states for this component